my-site.rb

```diff
@@ -2,7 +2,20 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```

Comment:
It is still a good idea to have a "root route" so that the user has somewhere to go when hitting the site by default